quinn/src/connection.rs

```diff
@@ -653,76 +713,12 @@ impl Read for BiStream {
             Err(UnknownStream) => Err(ReadError::UnknownStream),
         }
     }
-
-    fn stop(&mut self, error_code: u16) {
```

Comment:
Why was this removed?

Reply:
Because it was not `pub`, the compiler flagged it as unused (similar to `SendStream::reset()`). Added both back as `pub` methods.